CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.3.0]
```

Comment:
This should be underneath 1.2.0 as we haven't release 1.2.0 yet. Could you just rename that 1.2.0 heading to HEAD while you are at it please? In addition, could you refer to #87 and/or #77 in the changelog description?